docker/README.md

```diff
@@ -10,12 +10,63 @@ Follow the general installation instructions
 * [OSX](https://docs.docker.com/installation/mac/): [docker toolbox](https://www.docker.com/toolbox)
 * [Ubuntu](https://docs.docker.com/installation/ubuntulinux/)
 
-## Running the Container
+## Using CLI Version of LightGBM via Docker
 
-Build the container, for python users:
+Build a Docker image with LightGBM CLI:
 
+    ```
```

Comment:
I see in your forked repo codeblocks aren't properly rendered.
Please remove the indents (except the config example - it's rendered as needed).

![image](https://user-images.githubusercontent.com/25141164/32175449-13916a5e-bd96-11e7-871e-fcd2b56cbb6d.png)

Reply:
My bad, should be better now.

Reply:
Thank you!